index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Holliday Planner</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="nav-bar">
+        <img src="./picture/Logo.png" alt="brand-logo" width="80px" height="auto">
+        <div class="nav-menu">
+        <nav class="nav-links">
+            <ul>
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Find tours</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+        </div>
+        </div>
+        <h1>Holiday Planner</h1>
+        <div class="banner">
+            <div class="card">
+                <img src="./picture/image1.png" alt="logo">
+                <p><strong>Subhead 1</strong></p>
+                <p class="card-description">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Dolorum voluptates deserunt cumque quos, officiis sequi, iure in et totam minus animi aut unde numquam autem exercitationem itaque aspernatur ex temporibus.</p>
+            </div>
+            <div class="card">
+                <img src="./picture/image2.png" alt="logo">
+                <p><strong>Subhead 2</strong></p>
+                <p class="card-description">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Dolorum voluptates deserunt cumque quos, officiis sequi, iure in et totam minus animi aut unde numquam autem exercitationem itaque aspernatur ex temporibus.</p>
+            </div>
+            <div class="card">
+                <img src="./picture/image 3.png" alt="logo">
+                <p><strong>Subhead 3</strong></p>
+                <p class="card-description">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Dolorum voluptates deserunt cumque quos, officiis sequi, iure in et totam minus animi aut unde numquam autem exercitationem itaque aspernatur ex temporibus.</p>
+            </div>
+        </div>
+    <div class="section">
+        <p><strong>SECOND SUBSECTION</strong></p>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint excepturi nesciunt quibusdam nulla soluta. Totam, eligendi, nemo impedit eum repellat, ratione possimus debitis odit in sequi dicta aspernatur ab placeat!</p>
+    </div>
+    <div class="subsection2">
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Consequuntur veritatis blanditiis nisi doloremque quis iste repellat! Enim quam nobis itaque perferendis dolorem similique porro, rem saepe reiciendis. Quaerat, molestias sequi.</p>
+    </div>
+    <h2 id="contact">Contact</h2>
+    <div class="contact">
+        <div class="contact-form">
+            <form>
+                Email<input type="text" name="Email" id="email" style="background:url(email.svg) no-repeat"><br>
+                Message<br><textarea></textarea><br>
```

Comment:
avoid using br, should be used when it's necessary (ex. inside texts) here you should add display block for the textarea and/or label